service/storage_service.cc

```diff
@@ -6082,7 +6082,9 @@ future<> storage_service::stream_tablet(locator::global_tablet_id tablet) {
                 ranges_per_endpoint[r.host].emplace_back(range);
             }
             streamer->add_rx_ranges(table.schema()->ks_name(), std::move(ranges_per_endpoint));
+            slogger.debug("Streaming for tablet migration of {} started table={}.{} range={}", tablet, table.schema()->ks_name(), table.schema()->cf_name(), range);
```

Comment:
Is it intentional that the start is logged in `debug`, but finish logged in `info`? Shouldn't they be on the same level?

Reply:
It is intentional. It is too noise to add the start log. We do the same for the file stream.